test/integration/interactive-mode.test.ts

```
@@ -0,0 +1,332 @@
+      });
+    });
+  });
+
+  describe('Interactive Unsuitable Commands', function() {
+    const unsuitableCommands = [
+      'autocomplete',
+      'config',
+      'mcp'
+    ];
+
+    unsuitableCommands.forEach(cmd => {
```

ESLint

[error] Unexpected function call in describe block.

(mocha/no-setup-in-describe)